pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -179,7 +192,7 @@ let
         mkdir -p $out/Applications
         cp -R --no-preserve=mode,ownership ${browser}/Applications/${browserName}.app $out/Applications
         rm -f $out${browser.execdir or "/bin"}/${browserName}
-      '' + ''
+        '' + ''
```

Comment:
this still looks wrong, before was better

Reply:
what do you mean with before?

Reply:
```suggestion
      '' + ''
```